### ISSUE TITLE
1.5.2 is broken

### ISSUE THREAD

**Issue by dawiinci** [IndigoDomotics/indigo-pushover#25]
```
   Pushover Error                  Error in plugin execution InitializeMain:

Traceback (most recent call last):
IndentationError: ('expected an indented block', ('plugin.py', 120, 10, '    \t\tself.debugLog(u"Result: %s" % r.text)\n'))
```

Thanks for the quick fix for 1.5.1, but it fails to load completely.

**Comment by FlyingDiver** [comment]
Please test https://github.com/IndigoDomotics/indigo-pushover/releases/tag/1.5.3

**Comment by FlyingDiver** [comment]
I don't actually have Pushover set up so I can't test these...

**Comment by jcs** [comment]
If you don't use Pushover, maybe let others maintain this driver? The past few changes were merged and pushed out before I even had a chance to review them.

**Comment by FlyingDiver** [comment]
Yeah, I jumped the gun on this last fix.  Sorry about that.

**Comment by dawiinci** [comment]
Thank you though, it works now.

**Comment by FlyingDiver** [comment]
Good.  I'll let @jcs look it over in case there's anything else needs to be fixed before it gets pushed.

**Comment by jcs** [comment]
I would like to fix #26 but I think that is an upstream issue, so the 1.5.3 changes look ok to me.
